.contrib/Parser/DATAS/02 - Outdoor Zones/01 Kalimdor/Dustwallow Marsh.lua

```diff
@@ -547,6 +565,9 @@ root(ROOTS.Zones, m(KALIMDOR, {
 					["coord"] = { 41.8, 73.9, DUSTWALLOW_MARSH },
 					["timeline"] = { ADDED_4_0_3 },
 					["groups"] = {
+						objective(1, {	-- 0/1 Stonemaul Clan Avenged
```

Comment:
For this quest I remember having to use the quest item (a banner) in order to spawn Smolderwing.